__tests__/run_seed.test.js

```diff
@@ -405,4 +405,26 @@ describe("PATCH", () => {
       });
   });
 });
-
+describe("DELETE : status 204 and no content", () => {
+  test("Deletes given comment by comment_id", () => {
+    return request(app)
+      .delete("/api/comments/5")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get("/api/comments/5")
+          .expect(404)
+          .then(({ body: { msg } }) => {
+            expect(msg).toBe("Not found!");
+          });
+      });
+  });
```

Comment:
I really like the idea of testing that this comment no longer exists, however the endpoint get /api/comments/5 does not exist, but would still give the same 404 beacuse of your app.all. Did you instead mean to use the endpoint delete /api/commnents/5?